scripts/qa_crowbarsetup.sh

```diff
@@ -2496,6 +2496,13 @@ function custom_configuration
             if [[ $want_keystone_v3 ]] ; then
                 proposal_set_value keystone default "['attributes']['keystone']['api']['version']" "'3'"
             fi
+            if iscloudver 7plus && [[ $want_keystone_token_type ]]; then
```

Comment:
We don't need to hardcode any default value, better would be to use default from barclamp proposal.